pkg/operator/staticpod/controller/installer/installer_controller.go

```diff
@@ -452,27 +453,41 @@ func setAvailableProgressingNodeInstallerFailingConditions(newStatus *operatorv1
 
 	revisionStrings := []string{}
 	for _, currentRevision := range Int32KeySet(counts).List() {
-		count := counts[currentRevision]
-		revisionStrings = append(revisionStrings, fmt.Sprintf("%d nodes are at revision %d", count, currentRevision))
+		if count := counts[currentRevision]; count == 1 {
+			revisionStrings = append(revisionStrings, fmt.Sprintf("1 node achieved revision %d", currentRevision))
+		} else {
+			if count == numNodes {
+				revisionStrings = append(revisionStrings, fmt.Sprintf("all nodes achieved revision %d", currentRevision))
+			} else {
+				revisionStrings = append(revisionStrings, fmt.Sprintf("%d nodes achieved revision %d", count, currentRevision))
+			}
+		}
 	}
 	// if we are progressing and no nodes have achieved that level, we should indicate
 	if numProgressing > 0 && counts[newStatus.LatestAvailableRevision] == 0 {
-		revisionStrings = append(revisionStrings, fmt.Sprintf("%d nodes have achieved new revision %d", 0, newStatus.LatestAvailableRevision))
+		revisionStrings = append(revisionStrings, fmt.Sprintf("none of the nodes have achieved new revision %d", newStatus.LatestAvailableRevision))
 	}
 	revisionDescription := strings.Join(revisionStrings, "; ")
 
 	if numAvailable > 0 {
+		message := fmt.Sprintf("%d nodes are active; %s", numAvailable, revisionDescription)
```

Comment:
what does active mean?

Reply:
good question, I think "active" means "available" which means "they achieved the desired levels and are running" ?